janusgraph-test/src/test/java/org/janusgraph/diskstorage/util/MetricInstrumentedStoreManagerTest.java

```diff
@@ -0,0 +1,26 @@
+package org.janusgraph.diskstorage.util;
+
+import org.easymock.EasyMockExtension;
+import org.easymock.Mock;
+import org.janusgraph.diskstorage.BackendException;
+import org.janusgraph.diskstorage.keycolumnvalue.KeyColumnValueStoreManager;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+
+@ExtendWith(EasyMockExtension.class)
+public class MetricInstrumentedStoreManagerTest {
+
+    @Mock
+    private KeyColumnValueStoreManager mock;
+
+    @Test
+    public void testGetHadoopManagerReturnsBaseHadoopManager() throws BackendException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=5019203)